_includes/tutorials/rekeying-function/ksql/code/tutorial-steps/prod/send-to-api.sh

```diff
@@ -1,7 +1,7 @@
 tr '\n' ' ' < src/statements.sql | \
 sed 's/;/;\'$'\n''/g' | \
 while read stmt; do
-    echo '{"ksql":"'$stmt'", "streamsProperties": {}}' | \
+    jq --arg value1 "$stmt" -n '{ "ksql" : $value1 }'| \
```

Comment:
I'm not sure about jq.
We need to change rest of the tutorial for people to install all required dependencies

Reply:
We were already requiring it. This change adds a second usage in the same script. We can follow up in another issue to cover jq as a prerequisite.

Reply:
We use jq in the current template @ `templates/ksql/static/prod/code/send-to-api.sh`.

Reply:
Got it. Thanks